tests/ami/packages_sanity_tests.py

```diff
@@ -213,7 +254,15 @@ def main(
         test_type,
         random.randint(0, 1000),
     )
-    step = ScriptDeployment(rendered_template, timeout=120)
+    test_package_step = ScriptDeployment(
+        rendered_template, name="deploy.ps1", timeout=120
+    )
+
+    if os.path.exists(to_version) and os.path.isfile(to_version):
+        put_file_step = FileDeployment(to_version, "./ScalyrAgentInstaller.msi")
```

Comment:
I had a brief look at Libcloud code and you might not need to use Paramiko path workaround above if you use a full absolute path here (e.g. ``C:\\foo\\bar``), but would need to confirm.

And ``run()`` method onScriptDeployment might also need a small change to support windows style path separators (perhaps by adding path_separator argument to that class constructor or similar).